content/python/upgrade_virtual_guest_dedicated.py.md

```diff
@@ -0,0 +1,421 @@
+---
+title: "Working with upgrade virtual guest dedicated"
+description: "A few examples on interacting with upgrade virtual guest."
+date: "2019-03-27"
+classes:
+    - "SoftLayer_Virtual_Guest"
+    - "SoftLayer_Product_Order"
+    - "SoftLayer_Container_Product_Order_Virtual_Guest_Upgrade"
+tags:
+    - "virtualserver"
+---
+
+Upgrade a VSI(cpu, ram)
+```
+import json
+
+import SoftLayer
+
+client = SoftLayer.create_client_from_env()
+orderService = client['SoftLayer_Product_Order']
+
+# The id of the virtual guest you wish upgrade
+virtualGuestId = 11111
+
+# Package keyName.
+package_keyname = 'CLOUD_SERVER'
+
+object_mask = 'mask[prices]'
+
+# items KeyNames to be upgrade
+items = ['GUEST_PRIVATE_CORES_8', 'RAM_12_GB']
```

Comment:
I suggest you to add a sort explanation about how to retrieve that keys through getUpgradeItemPrices, maybe at the top (before all examples).

Reply:
As I explained above I think the others options "getItems" and "getItemPrices" are valid too. I will add short explanation as you recommended.